tests/units/test_units.py

```diff
@@ -193,3 +193,10 @@ def test_percent_units():
 def test_udunits_power_syntax():
     """Test that UDUNITS style powers are properly parsed and interpreted."""
     assert units('m2 s-2').units == units.m ** 2 / units.s ** 2
+
+
+def test_udunits_parentheses_and_power_syntax():
+    """Test that UDUNITS style parentheses usage is properly parsed and interpreted."""
+    assert units('kg(-1)').units == units.kg ** -1
+    assert units('W m(-2)').units == units.W * (units.m ** -2)
+    assert units('(W m-2 um-1)-1').units == (units.W ** -1) * (units.m ** 2) * units.um
```

Comment:
Would it be easy for you to turn this into a single parametrized test combined with the previous `test_udunits_power_syntax`?